src/components/icon/icon.pw.tsx

```diff
@@ -0,0 +1,538 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import { IconProps } from "components/icon";
+import { IconComponent, IconTooltipComponent } from "./component.test-pw";
+import { getDataElementByValue } from "../../../playwright/components/index";
+import { SIZE, COLOR, CHARACTERS } from "../../../playwright/support/constants";
+
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const colorData = [COLOR.ORANGE, COLOR.RED, COLOR.BLACK, COLOR.BROWN];
+
+test.describe("should check Icon component properties", () => {
+  test(`should check when aria-hidden prop is set as true`, async ({
```

Comment:
This test and the next one check whether the aria-hidden prop is true or false. They could be combined into one test.